routes/movies.js

```diff
@@ -53,6 +53,41 @@ router.get('/all', (request, response, next) => {
     )
 })
 
+router.get('/voting', (request, response, next) => {
+  Movie.find()
+    .then(data => {
+      /*
+        get an random index from the data array
+        add it to firstMovieIndex
+        do this for the second movie
+        repeat until firstMovieIndex and secondMovieIndex arent equal
+        return new array with both movies
+      */
```

Comment:
Do you need this comment somehow? I would remove it. :-)

Reply:
No, I dont need them. Wanted to make it easier to understand what I am trying to archieve here.

Already added in line 9: `// Comments will be deleted later on`